### DISCUSSION TITLE
Unknown column 'order_column'

### DISCUSSION THREAD

**Discussion by martin-ro** [freekmurze/eloquent-sortable#132] [Q&A] [answered]
`Column not found: 1054 Unknown column 'order_column' in 'field list' (SQL: select max(`order_column`) as aggregate from `course_categories`)`

I did

- `composer require spatie/eloquent-sortable`
- Added `.. implements Sortable` to the model
- Added `use SortableTrait;`
- Run migration + seeder

Did I miss something?

**Comment by freekmurze** [comment, marked as answer]
Make sure you have an `order_column` for the model.

**Reply by eddybrando** [reply]
This was easy to figure out for me as well, but to be honest, I had to look twice because other Spatie packages like [laravel-tags](https://github.com/spatie/laravel-tags) already include the new columns via migrations.

Maybe there could be a small addition to the README to make this clear from the get go, making it clear that one has to add those columns "manually" since they wont be added by the package.
